packages/manager/modules/vps/src/veeam/modal/schedule-backup/vps-schedule-backup.html

```diff
@@ -0,0 +1,26 @@
+<oui-modal
+    data-primary-label="{{ ::'vps_configuration_veeam_schedule_backup_action_confirm' | translate }}"
+    data-primary-action="$ctrl.onRescheduledAutomatedBackupConfirmed()"
+    data-primary-disabled="!$ctrl.model.rescheduleTimepicker"
+    data-secondary-action="$ctrl.onRescheduledAutomatedBackupCanceled()"
+    data-secondary-label="{{ ::'vps_configuration_veeam_schedule_backup_action_cancel' | translate }}"
+    data-on-dismiss="$ctrl.onModalCanceled()"
+    data-heading="{{:: 'vps_configuration_veeam_schedule_backup_title' | translate}}"
+    data-loading="$ctrl.isLoading"
+>
+    <p
+        class="oui-modal__text"
```

Comment:
`oui-modal__text` class doesn't exist. Looks like an old class that hasn't be removed everywhere. 
You can remove it safely.

Reply:
Cool, thanks.
Changed.